test/access/roles/PauserRole.test.js

```diff
@@ -1,9 +1,14 @@
 const { shouldBehaveLikePublicRole } = require('../../access/roles/PublicRole.behavior');
 const PauserRoleMock = artifacts.require('PauserRoleMock');
+const expectEvent = require('../../helpers/expectEvent');
 
 contract('PauserRole', function ([_, pauser, otherPauser, ...otherAccounts]) {
   beforeEach(async function () {
     this.contract = await PauserRoleMock.new({ from: pauser });
+    await expectEvent.inConstruction(this.contract, 'PauserAdded', {
```

Comment:
This should be a separate test (an `it`), and not in `beforeEach`. It should probably be part of `shouldBehaveLikePublicRole`, right?

Reply:
Good call